Upload file using azcopy

msdz opened the pull request:
Specifies whether the destination blob is a block blob, a page blob, or an append blob. This option is applicable only when you are uploading a blob. Otherwise, an error is generated. If the destination is a blob and this option is not specified, by default, AzCopy creates a block blob.

azurecla commented:
Hi __@harriha__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tysonn commented:
@ningkuang - Please review this contribution and let me know if I can merge it.

NingKuang commented:
Yes, I reviewed them, looks good to me. Thank you!

Commit:
Upload file using azcopy

Specifies whether the destination blob is a block blob, a page blob, or an append blob. This option is applicable only when you are uploading a blob. Otherwise, an error is generated. If the destination is a blob and this option is not specified, by default, AzCopy creates a block blob.

cynthn reviewed (changes requested):
Thank you for the feedback! I agree with adding the note about needing to specify the blob type if you use AzCopy, but I don't want to add the 2 code blocks with that syntax into this article. Can you remove everything except the "Important" note? Thank you!
